brave/src/main/java/brave/propagation/B3Propagation.java

```diff
@@ -73,6 +130,8 @@
     this.fields = Collections.unmodifiableList(
```

Comment:
Does this need to be updated?

Reply:
yes. good eye.. though this change invalidates this slightly as almost always you will have all fields possible (if any injector has multi..)

Reply:
adds a lot of code to be precise here, heh